R/convert_se_to_raw_data.R

```diff
@@ -20,14 +20,34 @@ convert_mae_to_raw_data <- function(mae) {
   
   data <- gDRutils::MAEpply(mae, convert_se_to_raw_data)
   
+  if ("swap_sa" %in% names(merged_df)) {
```

Comment:
Do we have a unit test for this case?

Reply:
Sorry, this chunk of code has been remove locally, I updated the PR